src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditor.java

```diff
@@ -1192,14 +1192,15 @@ public void scrollToY(int y, int animateMs)
       fireEvent(new ScrollYEvent(Position.create(getFirstVisibleRow(), 0)));
    }
 
-   public void insertCode(String code)
+   public void insertCode(String code, boolean unused)
```

Comment:
We don't have like a naming convention for this in Java, like `_unused` ?

Reply:
As far as I know, we don't, at least for the Java sources.